packages/terra-application-layout/src/header/ApplicationHeader.scss

```diff
@@ -0,0 +1,34 @@
+:local {
+  .application-header {
+    background-color: var(--terra-application-header-background-color, #1b7fcd);
+    border: var(--terra-application-header-border, 0.07143rem solid rgba(255, 255, 255, 0.17));
+    height: var(--terra-application-header-height, 3.5725rem);
+    position: relative;
+  }
+
+  .toolbar-toggle {
+    align-items: center;
+    border-right: var(--terra-application-header-toggle-border-right, 0.07143rem solid rgba(255, 255, 255, 0.17));
+    box-shadow: var(--terra-application-header-toggle-box-shadow, inset -0.07143rem 0 0 0 rgba(0, 0, 0, 0.17));
+    height: 100%;
+    width: var(--terra-application-header-toggle-width, 3.5725rem);
+  }
+
+  .toggle-button {
+    background-color: var(--terra-application-header-toggle-background-color, transparent);
+    border: hidden;
```

Comment:
`hidden` is not a valid border style, maybe you meant `border: 0;` or `border: none;`?

Reply:
This confused me too. Axe was complaining that 'none' wasn't a valid border value, but it accepts 'hidden'. I can come back around on this and investigate more.

Reply:
Might try setting `border: 0` instead and see if aXe complains about that style.

Reply:
That's actually what @dkasper-was-taken did, and it seems to work fine 👍